src/fundus/publishers/de/waz.py

```diff
@@ -44,3 +45,9 @@ def topics(self) -> List[str]:
             authors = generic_author_parsing(self.precomputed.meta.get("author"))
             topics = generic_topic_parsing(self.precomputed.meta.get("keywords"))
             return [topic for topic in topics if topic not in authors]
+
+    class V1_1(V1):
+        VALID_UNTIL = datetime.date.today()
+        _paragraph_selector = CSSSelector(".article-body > p")
```

Comment:
This falsely also extracts the summary a second time

Reply:
Done.